modules/backend/helpers/Backend.php

```diff
@@ -156,4 +157,54 @@ public function dateTime($dateTime, $options = [])
 
         return '<time'.Html::attributes($attributes).'>'.e($defaultValue).'</time>'.PHP_EOL;
     }
+
+    /**
+     * Decompiles the compilation asset files
+     *
+     * This is used to load each individual asset file, as opposed to using the compilation assets. This is useful only
+     * for development, to allow developers to test changes without having to re-compile assets.
+     *
+     * @param string $file The compilation asset file to decompile
+     * @param boolean $skinAsset If true, will load decompiled assets from the "skins" directory.
+     * @return array
```

Comment:
Missing an @throws Exception

Also, it might be better to make specific exceptions for specific errors.

See for example: https://blog.nikolaposa.in.rs/2016/08/17/exceptional-behavior-best-practices/

It makes it possible to catch the correct "error".

Reply:
@nathan-van-der-werf Good idea, will implement. :)

Reply:
I've added `Backend\Helpers\Exception\DecompileException` - it's not unique to each individual exception, but it works with our error handler at least.